README.md

```diff
@@ -49,15 +49,15 @@ project.ext.react = [
 To configure your project to use haul, run the following:
 
 ```bash
-yarn run haul init
+yarn haul init
 ```
 
 This will automatically add the configuration needed to make Haul work with your app, e.g. add `webpack.haul.js` to your project, which you can customise to add more functionality.
 
 Next, you're ready to start the development server:
 
 ```bash
-yarn run haul start -- --platform ios
+yarn haul start -- --platform ios
```

Comment:
That extra `--` is also not needed since Yarn 1.0

Reply:
Some of us still use < 1.0 :D